src/main/webapp/WEB-INF/tags/instructor/course/courseEditInstructorListPanelBody.tag

```diff
@@ -16,17 +16,30 @@
         <input type="hidden" name="<%=Const.ParamsNames.USER_ID%>" value="${data.account.googleId}">
     
         <div id="instructorTable${instructorPanel.index}">
-            <c:if test="${not empty instructorPanel.instructor.googleId}" >
-                <div class="form-group">
-                    <label class="col-sm-3 control-label">Google ID:</label>
-                    <div class="col-sm-9">
-                        <input class="form-control immutable" type="text" id="<%=Const.ParamsNames.INSTRUCTOR_ID%>${instructorPanel.index}"
-                                value="${instructorPanel.instructor.googleId}" maxlength="<%=FieldValidator.GOOGLE_ID_MAX_LENGTH%>" 
-                                tabindex="3" disabled>
+            <c:choose>
+                <c:when test="${not empty instructorPanel.instructor.googleId}">
+                    <div class="form-group">
+                        <label class="col-sm-3 control-label">Google ID:</label>
+                        <div class="col-sm-9">
+                            <input class="form-control immutable"
+                                    type="text" id="<%=Const.ParamsNames.INSTRUCTOR_ID%>${instructorPanel.index}"
+                                    value="${instructorPanel.instructor.googleId}"
+                                    maxlength="<%=FieldValidator.GOOGLE_ID_MAX_LENGTH%>" tabindex="3" disabled>
+                        </div>
                     </div>
-                </div>
-            </c:if>
-        
+                </c:when>
+                <c:otherwise>
+                    <div class="form-group">
+                        <label class="col-sm-3 control-label">Google ID:</label>
+                        <div class="col-sm-9">
+                            <p class="form-control-static">
+                                <span class="text-warning">Not available. Instructor is yet to join this course.</span>
+                            </p>
```

Comment:
Is it possible to change `<c:when>` and `<c:otherwise>` to only cover the `<input>` element or the `<p>` element? The other elements are exactly the same, and do not need to be put in a conditional.